go/client/cmd_simplefs.go

```diff
@@ -44,14 +44,17 @@ func NewCmdSimpleFS(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comm
 	}
 }
 
+const mountDir = "/keybase"
+
 func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
-	mountDir := "/keybase"
 
 	path = filepath.ToSlash(path)
 	if strings.HasSuffix(path, "/") {
 		path = path[:len(path)-1]
 	}
```

Comment:
`filepath.Clean` can take care of this, and it gets called automatically by `Abs` and `EvalSymlinks`.